src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1018,7 +1018,7 @@ public Settings additionalSettings() {
             return settingsFilter;
         }
 
-        settingsFilter.add("opendistro_security.*");
```

Comment:
any downside of keeping `"opendistro_security.*"` in the filter?

Reply:
+1 If there are any settings which are from opendistro they would be filtered out. 
I think we should keep it.

Reply:
Will add this line back.